planetcantile/data/generate.py

```diff
@@ -50,10 +50,10 @@ class Tmsparam(object):
     extent: tuple[float, float, float, float]
     # Tile Matrix Set coordinate reference system
     crs: CRS
-    # Width of each tile of this tile matrix in pixels (default is 256).
-    tile_width: int = 256
-    # Height of each tile of this tile matrix in pixels (default is 256).
-    tile_height: int = 256
+    # Width of each tile of this tile matrix in pixels (default is 512).
+    tile_width: int = 512
```

Comment:
Why the swap here? To match something else?

Reply:
yes, rio-cogeo uses 512x512 tiles which is also the default in the COG GDAL driver (https://gdal.org/drivers/raster/cog.html), there is some ongoing discussion in #6 related to this. The good news for some is that the larger tiles would reduce the number network requests by a factor of 4 per zoom level.

Reply:
also glad to see others are interested in these products @drferling, none of these tms specs are finalized yet so hopefully you aren't using them yet!

Reply:
:+1: I just pulled my notes up for doing a Titiler deploy in our AWS environment, so I'm not using these yet. My notes are reminding me that the deploy was a pain (normal permissions friction, not a struggle due to the code base). 

I am hoping to get this all deployed shortly, though, so hopefully using these quite soon.